platform/platformsh.css

```diff
@@ -47,7 +47,7 @@ body.skin-blue {
 }
 
 body {
-  background-color: var(--background);
+  background-color: rgb(240, 242, 245);
```

Comment:
This isn't one of our brand colors, is it?  I want to avoid magic numbers in the CSS if at all possible.

Reply:
this was another of my UI references, sorry, I reverted this change and remove/replaced any UI color updates I did